src/Orleans.Streaming.Redis/Storage/RedisDataManager.cs

```diff
@@ -147,6 +160,8 @@ public Task<IEnumerable<RedisValue>> GetQueueMessagesAsync(int count)
                     items.Add(item);
                 }
 
+                _logger.Debug("Retrieved {Count} messages", items.Count);
```

Comment:
Should `IsEnabled` be used to omit these invocations? I'm honestly not sure of any of the overhead associated with those conditionals etc to avoid a log method...particularly since I'm unsure of how much work Serilog does before concluding "nothing wants this message".

Reply:
This message should be called periodically enough that it shouldn't matter too much. I second the overhead concerns and not knowing enough about serilog's underlying implementation.

Reply:
https://github.com/serilog/serilog/blob/dev/src/Serilog/Core/Logger.cs#L366

Seems like all log calls are gated by it - so at most it would eliminate a single function call. Which they themselves could avoid by not internally calling IsEnabled.........

Reply:
Good find! That does seem to confirm that very little processing occurs.